### ISSUE TITLE
The value of the loss function increase during the training

### ISSUE THREAD

**Issue by abdou31** [yinguobing/cnn-facial-landmark#84]
Hello Yin, 
As you know for the first model that I have created with this CNN, it worked perfectly but for the face oriented ( profile face), the tracking is lost and I have discovered that the problem is coming from the false annotations.
After some research, I have found a solution, So I have organized a new dataset and I have annotated this dataset using OpenFace, for now, the dataset is correctly annotated.
I followed all the steps to create the three files validation.record, train.record and test.record
I have started the training, after some steps (60 000), I stop the training and I have seen the graph of the loss function using Tensorboard.
The result was so bad as you can see:

![image](https://user-images.githubusercontent.com/19480228/67246141-edc8b380-f45d-11e9-8452-ebd74bcd8d4b.png)

comparing to the old model:

![graph](https://user-images.githubusercontent.com/19480228/67246211-1d77bb80-f45e-11e9-94ea-423de6ee9d04.PNG)

I would like what is the problem here?
How can I correct the accuracy of the prediction?

Note: I have used the old version of CNN facial landmark project

**Comment by yinguobing** [comment]
That's hard to tell.
Try reducing the learning rate first.

**Comment by abdou31** [comment]
What value will be good for `learning_rate`?
What if I set it for 0.0001 ?

**Comment by abdou31** [comment]
After reducing the `learning_rate` and re-split the dataset  random( give 140000 images to train instead of 150000 images).
I visualize the loss graph and that was the result:
![image](https://user-images.githubusercontent.com/19480228/67716307-db092e00-f9cb-11e9-9ae6-81ea3cfa1aff.png)

I can tell anything before testing the model on images and videos

**Comment by abdou31** [comment]
I solved the problem by following the two points that I have mentioned in the previous comment.

**Comment by ZlaaM** [comment]
@abdou31 
Is the new dataset you used 68 landmarks?
if yes please I need it

**Comment by abdou31** [comment]
NoI used the same dataset but with different number of annotation landmarks

**Comment by ZlaaM** [comment]
@abdou31 thank you

**Comment by ZlaaM** [comment]
@abdou31 
How much value of train steps and learning rate do I have to choose to get good results?

**Comment by abdou31** [comment]
@ZlaaM  
I have set 150 000 steps and for learning_rate=1e-3
